Makefile

```diff
@@ -169,6 +171,31 @@ else
             $(TESTS)
 endif
 
+jstest: TESTS ?= UI/tests
```

Comment:
I understand why you would do this, but how then do we prevent the tests from ending up on production deployments? Also' everything in UI is accessible from outside the server. I'd prefer to put the tests elsewhere so it's easier for packagers and the like to exclude tests from installation.

Reply:
Jest Tests sources can be anywhere, so we can remove them from UI